test.sh

```diff
@@ -7,45 +7,21 @@ OS=${OS:="centos"}
 OS_VERSION=${OS_VERSION:="7"}
 PYTHON_VERSION=${PYTHON_VERSION:="2"}
 ACTION=${ACTION:="test"}
-IMAGE="$OS:$OS_VERSION"
-
-# Optionally specify repo and branch for osbs-client to test changes
-# which depend on osbs-client patches not yet available in upstream master
-OSBS_CLIENT_REPO=${OSBS_CLIENT_REPO:-https://github.com/containerbuildsystem/osbs-client}
-OSBS_CLIENT_BRANCH=${OSBS_CLIENT_BRANCH:-master}
+IMAGE=${IMAGE:="$OS:$OS_VERSION"}
```

Comment:
This should be dynamic, i.e. use ruby for mdlint here

Reply:
I don't want to guess which image to run when I want to run linter locally. I want just specify action and enjoying output. IMAGE shall be specified in test.sh for markdownlint

Reply:
I see, OK.